projects/wvr-elements/src/lib/shared/config.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ConfigService {
+  baseUrl: string;
```

Comment:
I feel a better approach would be for a type of interface to define the configuration and have it injectable as apposed to injecting the config service in order to get a configuration value.

Reply:
The responsibility of the config service likely should only be to fetch the configurations on runtime loading of angular. Having it double as the actual configuration values is not ideal.

Reply:
This issue https://github.com/orgs/TAMULib/projects/4#card-41868508 will introduce changes to this effect.